Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'grpc', '>= 1.53.0'
 gem 'crawler_detect'
 
 # Serving requests
-gem 'puma', '~> 4.3'
+gem 'puma', '~> 5.6'
```

Comment:
I've updated puma to the the smallest version that has all CVEs fixed.
Puma is usually, IME, easy to upgrade so maybe this was behind for some specific and know reason.
If that is the case I can revert this upgrade the others.